openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -150,8 +150,13 @@ jQuery(function () {
             .then((module) => module.initEdit());
     }
     if (document.getElementsByClassName('imageIntro').length) {
-        import(/* webpackChunkName: "book_cover_manage" */ './cover_add_manage')
-            .then(module => module.initAddCoverImport());
+        import(/* webpackChunkName: "covers_add_manage" */ './covers_add_manage')
```

Comment:
Since both concern the same feature (covers) let's keep the JS in a single file.

I believe the check for `document.getElementsByClassName('imageIntro').length` also works for the below code.

Inside `initCoversAddManage` we can do the check for `document.getElementsByClassName('manageCovers').length`.

Reply:
After a few days of testing and trying to figure this one out, I haven't been able to make it work. For whatever reason, it appears the addition of iframes to the DOM are not registered for both the `Document` and `JQueryStatic` objects inside this file. The only way I could make jQuery or regular JavaScript find the elements needed to be operated on inside the iframes was to call the JavaScript for the iframes inside of index.js. Let me know if you have any other suggestions though. I just couldn't seem to make the above work, nor any other solutions I tried looking for on the internet. Sorry!

Reply:
I was able to make this one work, but I still had to do imports in index.js.

Reply:
This is perfect. Thanks!